documentation/homework/week1/src/components/todoListItem.js

```diff
@@ -0,0 +1,24 @@
+import React,{Component} from 'react';
+
+class TodoListItem extends Component {
+    render() {
+        const id = this.props.id;
+        const description = this.props.description;
+        const deadline = this.props.deadline;
```

Comment:
You can use destructing here `const { id, description, deadline } = this.props;`